.github/workflows/test-ballerine-wf.yml

```diff
@@ -0,0 +1,57 @@
+name: Publish workflows-service image
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      # Run this pipeline only if there are changes in specified path
+      - "services/workflows-service/**"
+    branches:
```

Comment:
One we pushed it to dev and ran e2e test on this env,
The same result should be placed when running on others as well because we promote the image tag to another env

Reply:
Ohh ok got it then the changes should be a little different . I will address it